AirCasting/SessionViews/SessionLoadingView.swift

```diff
@@ -0,0 +1,30 @@
+// Created by Lunar on 24/06/2021.
+//
+
+import SwiftUI
+
+struct SessionLoadingView: View {
+    var body: some View {
+        HStack {
+            Image("ABLoading")
+                .resizable()
+                .frame(width: 60, height: 60)
+            VStack(alignment: .leading) {
+                Text("Your AirBeam is gathering data.")
```

Comment:
do we have this static `Strings` struct on `development` branch? If yes we should put such string over there (applies to all literal strings, I wont point those in further comments)

Reply:
Changed :)